### DISCUSSION TITLE
Cuál extensión de Markdown recomiendan?

### DISCUSSION THREAD

**Discussion by DanielOe21** [LaunchX-InnovaccionVirtual/CursoIntroPython#39] [Q&A] [answered]
@FernandaOchoa comentó en el live del lunes que en su VS Code tiene instalada una extensión para la previsualización de Markdown pero no recordaba su nombre,,, ¿Alguien tiene alguna idea / sugerencia?

**Comment by DamianGonzalezV** [comment, marked as answer]
La extension es Office Viewer(Markdown Editor)

**Reply by DanielOe21** [reply]
¿Es la q usa Fer?

**Reply by DamianGonzalezV** [reply]
si

**Reply by FernandaOchoa** [reply]
Sí! Muchas Gracias!!! No la recordaba, debe ser la edad...
